$NetBSD: patch-aa,v 1.3 2025/12/30 15:31:48 leot Exp $

stdarg(3) is required by various functions that can be called with variable
argument lists.

--- src/intrfn.h.orig	2022-04-10 09:22:11.000000000 +0000
+++ src/intrfn.h
@@ -41,6 +41,8 @@ extern "C" {
 #include <curses.h>
 #endif
 
+#include <stdarg.h>
+
 void aff_copy(WINDOW *window);
 void aff_copy_short(WINDOW *window);
 
